ui/src/experiments/ExperimentsLandingPage.js

```diff
@@ -68,7 +68,7 @@ const ExperimentsLandingPage = () => {
         which results in incorrect /experiments/experiments prefix.
          */}
           <Route
-            path="experiments"
```

Comment:
Just curious, how come removal of this didn't show any error previously? https://github.com/caraml-dev/xp/pull/45/files#diff-557155280a10fe2c6a10a1b299138e6f690220fb5ecffb8546ce3aa459a57287L71

Reply:
The problem is noticed when we are using the context menu to switch from one nested section to Experiments (eg: /segments/{id} to /experiments). I missed this check when testing locally previously.